SPEC_MAINTAINER_PROTOCOL.md

```diff
@@ -0,0 +1,80 @@
+# Spec Maintainer Protocol
+
+> We have a protocol to maintain specs of protocols! How meta, but very useful.
+
+## Motivation
+
+The protocol specifications should be treated as the ultimate source of truth for how the protocol works, how to implemented and what users and builders can expect on how it behaves. However, writting specs for a protocol is not novelty and there has been multiple decades of research on how to do this well, from having a reference implementation, standards bodies and/or formal verification to ensure both code and specs behave the same. It is a hard problem and ultimately one that questions how programming is done as a whole.
+
+Understanding this, we want to make sure to provide a platform in which the Core Developers and the whole IPFS community can collaborate and build specs and code that as much of a pair as possible. This compromises into two main chunks of work:
+
+- Writting the spec and iterating on it to the point that the spec and the code progress smoothly together.
+- For mature and more stable specs, have tests (using [Protocol Driven Development](https://github.com/ipfs/pdd), [TLA+](https://lamport.azurewebsites.net/tla/tla.html) or another framework of verification) to ensure that all implementations stand correct.
+
+The very aspirational future is to have a spec language in which we can describe how the program should function and it will automatically create the tests to verify it. The aspiration future ++ is having such spec language that writes the code itself and verifies it 🚀.
+
+The Spec Maintainer Protocol takes a large amount of inspiration from the repo [Lead Maintainer Protocol](https://github.com/ipfs/team-mgmt/blob/master/LEAD_MAINTAINER_PROTOCOL.md).
+
+## Maintaining a Specification
+
+A Specification Maintainer is a contributor that has shown that understand the subsystem of the spec that is maintaining and that is passionate about documenting it well. There is a benefit for this contributor to be actively involved with the development of the subsystem at least in one or more of the languages.
+
+Each spec should be maintained at least by 1 person and up to 3 for efficiency purposes.
+
+The spec maintainer(s) are recognized at the top of the specification document on the list "Maintainers(s)" and these are the people that should be requested to review a PR to that specific spec.
+
+### Responsibilities
+
+As a Spec Maintainer, you are expected to:
+
+- Respect and follow the [IPFS Code of Conduct](https://github.com/ipfs/community/blob/master/code-of-conduct.md).
+- Have a great understanding of the subsystem  purpose and how it is used by other parts of the project.
+- Review and merge PRs to the Spec
```

Comment:
Specs are not code and changes are hard to revert. There should be a review process for every non-trivial change (even if everyone in the process just gives a :+1:).

I'd expect a maintainer to merge cleanups/corrections, shepard discussions, and bring proposals to the attention of the IPFS Spec WG (or something like that).

Reply:
I would go as far as to say you don't revert a spec. Once someone has built on top, that is it, that spec is a thing, there is no taking it back, there is only moving forward.

Reply:
Hmm I now see @Stebalien that you say essentially the same thing in another [comment](https://github.com/ipfs/specs/pull/214#discussion_r303688494)